### PR TITLE
Gitlab 15.4 Updates

### PR DESCRIPTION
**Pull request by DarwinJS**
*Issue #132 

*Description of changes:*
- Get Gitlab 15 Working

Additional Todos:

- [x] End to end test for 15.x with updated automation
- [ ] Counter-test cipher fix on 14.x and maybe 13.x

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .taskcat.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ project:
   parameters:
     RemoteAccessCIDR: 127.0.0.1/32
     QSS3BucketName: $[taskcat_autobucket]
-    QSS3BucketRegion: $[taskcat_current_region]
+    QSS3BucketRegion: us-east-1
```
Comment (tlindsay42) [inline .taskcat.yml:8]: @troy-ameigh ^^, Darwin implemented this to workaround an issue where the taskcat container can only handle buckets in us-east-1.

@DarwinJS Did you create an issue for this? Didn't see it in the taskcat repo or else I'd tag it in this PR.

File: .taskcat.yml [hunk 2]
```diff
@@ -79,24 +78,26 @@ tests:
     regions:
     - ap-south-1
     - us-east-1
+    -
```
Comment (tlindsay42) [inline .taskcat.yml:81]: Was this intentional? Likely need to remove. ?

File: .taskcat.yml [hunk 3]
```diff
@@ -79,24 +78,26 @@ tests:
...
       EKSPublicAccessEndpoint: Disabled
       ProvisionBastionHost: Enabled
-      DomainName: qsgsmg3.devops4the.cloud
       CreateSslCertificate: 'Yes'
       SslCertificateIssuerEmail: gitlab@$[taskcat_random-string].com
       ConfigureContainerInsights: 'Yes'
       ConfigureGrafana: 'Yes'
-      PublicDNS: CreateNew
+      PublicDNS: UseExisting
+      PublicHostedZoneId: Z08828103MVPQNHL7GWYE
```
Comment (tlindsay42) [inline .taskcat.yml:99]: Please replace with an override (example: `PublicHostedZoneId: --override--`). Any input parameter value can be overridden via a `.taskcat_overrides.yml` file, but it's best practice to explicitly list override or something to that effect for developer-specific and sensitive values. It's also not best practice to commit live resource IDs.

**Review by tlindsay42 — CHANGES_REQUESTED** [review]

File: .taskcat.yml (hunk 1 above)
Comment (DarwinJS) [reply .taskcat.yml:8]: Here you go: https://github.com/aws-ia/taskcat/issues/764

File: .taskcat.yml (hunk 2 above)
Comment (DarwinJS) [reply .taskcat.yml]: Nope and guess what - taskcat linting caught this one!

**Comment by DarwinJS** [comment]
@tlindsay42 - ready for merge when you are.

**Review by tlindsay42 — APPROVED** [review]

**Pull request merged**